test/tests/functional/pbs_server_periodic_hook.py

```diff
@@ -384,9 +384,8 @@ def test_other_pbs_operations_work(self):
         a = {'Resource_List.select': '1:ncpus=1',
              'Resource_List.walltime': 3}
         j = Job(TEST_USER, attrs=a)
-        j.set_sleep_time(3)
+        j.set_sleep_time(10)
```

Comment:
You are still submitting the job with a walltime of 3s.  PBS will kill the job.  In any case, 3s is still probably OK because you just want the job to end.  It doesn't matter if a slow machine pauses since you are no longer trying to catch the job in the 'R' state.

Reply:
Yes, I agree. Thanks for catching it @bhroam